### ISSUE TITLE
RandomTweaker breaking `AstralSorcery` crystal attunment

### ISSUE THREAD

**Issue by Krutoy242** [Project-RT/RandomTweaker#76]
When `RandomTweaker-1.4.2` installed, crystals cant be attuned to constellations:
![image](https://github.com/Project-RT/RandomTweaker/assets/936618/f0e20494-f0d7-4e33-a5cd-d352f167a158)

When mod removed, everything working fine:
![image](https://github.com/Project-RT/RandomTweaker/assets/936618/f2d76144-1a4e-4811-a00b-9375799d7641)

`attunementModification` config set to `false`
```cfg
astralsorcery {
    # If true, Modify attunement and add related events. [default: false]
    B:attunementModification=false
}
```

Setting this config option to `true` fixes the issue.

**Comment by XHL315** [comment]
badly, I will try to fix it

**Comment by Krutoy242** [comment]
Oh sorry, i didn't knew I could close issues with commit comments in other repositories.

**Comment by XHL315** [comment]
fix, see 1.4.3
